CHANGELOG.md

```diff
@@ -15,6 +15,13 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+# 5.14.1
```

Comment:
We just released 5.14.1 so bumping this
```suggestion
# 5.14.2
```

Reply:
Hi @heath-freenome , I just committed the requested change.